### PR TITLE
Add rector and drop PHP 7.3

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Subject

I am targeting this branch, because {reason}.

Closes #{put_issue_number_here}.

## Changelog

```markdown
### Removed
- Support for PHP 7.3
```

### CODE REVIEW

File: src/Writer/GsaFeedWriter.php [hunk 1]
```diff
@@ -20,32 +20,15 @@
  */
 final class GsaFeedWriter implements WriterInterface
 {
-    public const LIMIT_SIZE = 31457280; // 30MB
+    public const LIMIT_SIZE = 31_457_280; private \SplFileInfo $folder;
```
Comment (VincentLanglet) [inline src/Writer/GsaFeedWriter.php:23]: Weird

File: src/Writer/SitemapWriter.php [hunk 2]
```diff
@@ -18,28 +18,16 @@
  */
 final class SitemapWriter implements WriterInterface
 {
-    public const LIMIT_SIZE = 10485760;
+    public const LIMIT_SIZE = 10_485_760;
```
Comment (VincentLanglet) [inline src/Writer/SitemapWriter.php:21]: Weird

File: src/Writer/XlsxWriter.php [hunk 3]
```diff
@@ -26,35 +26,17 @@
...
-     * @var Spreadsheet
-     */
-    private $spreadsheet;
+    private ?Spreadsheet $spreadsheet = null;
 
-    /**
-     * @var Worksheet
-     */
-    private $worksheet;
+    private ?Worksheet $worksheet = null;
```
Comment (jordisala1991) [inline src/Writer/XlsxWriter.php:37]: Is correct to change both properties to nullable?

Comment (VincentLanglet) [reply src/Writer/XlsxWriter.php:37]: The property is not initialized in the constructor so far.
Phpstan is complaining:
```
Class Sonata\Exporter\Writer\XlsxWriter has an uninitialized property $spreadsheet. Give it default value or assign it in the constructor.
```
If you call `write()` without `open()` before, you'll end with an access to an uninitialized property.

Later we will need to add some null check and throw some exceptions.

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**